helm/cfgov/values.yaml

```diff
@@ -233,6 +233,7 @@ postgresql:
   enabled: false
 
 elasticsearch:
+  test: false
```

Comment:
The correct value is `tests`.

See [here](https://github.com/elastic/helm-charts/blob/main/elasticsearch/templates/test/test-elasticsearch-health.yaml#L1).